code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -1044,46 +1043,41 @@
 	reac_volume = round(reac_volume,0.1)
 	exposed_turf.acid_act(acidpwr, reac_volume)
 
+// when you ingest acid (not the fun kind) death awaits... 
+/datum/reagent/toxin/acid/on_mob_life(mob/living/carbon/victim, delta_time, times_fired)
+	var/acid_dmg = acidpwr * 0.1 // results in 1 for sulphric acid, 4.2 for flurosuluric acid, and 0.5 for nitric acid 
+	victim.adjustBruteLoss(acid_dmg * REM * delta_time, 0)
+	victim.adjustFireLoss(acid_dmg * 2 * REM * delta_time, 0) // take double burn damage
+
+	if(DT_PROB(16, delta_time))
+		victim.adjustOrganLoss(ORGAN_SLOT_STOMACH, acid_dmg * REM * delta_time)
+		victim.emote("gurgle")
+	else if(DT_PROB(16, delta_time))
+		victim.adjustOrganLoss(ORGAN_SLOT_LIVER, acid_dmg * REM * delta_time)
+		victim.emote("grimace")	
+	..()
+	//return TRUE  (is this needed?)
+
 /datum/reagent/toxin/acid/fluacid
 	name = "Fluorosulfuric Acid"
 	description = "Fluorosulfuric acid is an extremely corrosive chemical substance."
 	color = "#5050FF"
 	creation_purity = REAGENT_STANDARD_PURITY
 	purity = REAGENT_STANDARD_PURITY
-	toxpwr = 2
 	acidpwr = 42.0
 	ph = 0.0
 	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
 
-// SERIOUSLY
-/datum/reagent/toxin/acid/fluacid/on_hydroponics_apply(obj/item/seeds/myseed, datum/reagents/chems, obj/machinery/hydroponics/mytray, mob/user)
-	. = ..()
-	if(chems.has_reagent(type, 1))
-		mytray.adjust_plant_health(-round(chems.get_reagent_amount(type) * 2))
-		mytray.adjust_toxic(round(chems.get_reagent_amount(type) * 3))
-		mytray.adjust_weedlevel(-rand(1,4))
-
-/datum/reagent/toxin/acid/fluacid/on_mob_life(mob/living/carbon/M, delta_time, times_fired)
-	M.adjustFireLoss((current_cycle/15) * REM * normalise_creation_purity() * delta_time, 0)
-	. = TRUE
-	..()
-
 /datum/reagent/toxin/acid/nitracid
 	name = "Nitric Acid"
 	description = "Nitric acid is an extremely corrosive chemical substance that violently reacts with living organic tissue."
 	color = "#5050FF"
 	creation_purity = REAGENT_STANDARD_PURITY
 	purity = REAGENT_STANDARD_PURITY
-	toxpwr = 3
 	acidpwr = 5.0
 	ph = 1.3
 	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
 
-/datum/reagent/toxin/acid/nitracid/on_mob_life(mob/living/carbon/M, delta_time, times_fired)
-	M.adjustFireLoss((volume/10) * REM * normalise_creation_purity() * delta_time, FALSE) //here you go nervar
-	. = TRUE
```

Comment:
Why are you making the complicated to craft nitric acid worse than it's simpler precursor? This acid was designed with the intent of being a anti-carbon acid rather than fluorosulfuric which is better for wrecking items and objects.

Reply:
Just because it's more complicated doesn't mean it should be more deadly.  Nitric is used more as a crafting recipe to get hard to obtain chems and explosives and is the weakest acid based on the ph scale.

Reply:
You should probably change the description of nitric acid then, considering it had no additional effects on carbons anymore, and is just a worse version of flurosulfic acid now.